docs/src/tutorials/conversions_views.md

```diff
@@ -507,3 +507,33 @@ Base.ReshapedArray{RGB{Normed{UInt8,8}},3,Base.ReinterpretArray{RGB{Normed{UInt8
 While there is little or no performance cost to making use of
 JuliaImage's convenient views, sometimes the types can get
 complicated!
+
+## Preserving colorant type while conversion
+
+While using Float32, Float64 to convert the raw storage type of an image 
+the color space information is lost. Replacaing Float32, Float64 with their lower-case
+forms preserves the colorant type.
```

Comment:
I think it would be better to present this as an example of how you control color space and element type generally. There are two orthogonal concepts:

- what colorspace do you want: `RGB`, `HSV`, `Gray`, or pure numeric?
- what precision of storage type do you want: `Float32`, `N0f8`, etc.?

The lowercase `float32`, `n0f8` are designed to control the second *without modifying the first*. In contrast, `Gray.(img)` and `HSV.(img)` (*without* the eltype specification) control the first without (as far as possible) modfiying the second. And finally, `Gray{N0f8}.(img)` and `HSV{Float32}.(img)` (*with* the eltype specification) control both simultaneously.

Limiting your explanation to grayscale also misses out on the opportunity to present the broader perspective on color spaces in general, and might make the distinction between `float32` and `Float32` seem like hair-splitting. The importance of the distinction becomes much clearer in the context of color images. I'd also recommend demoing the difference between `float32` and `Float32` applied to an RGB image: while for grayscale you have
```julia
julia> img = testimage("cameraman");

julia> summary(img)
"512×512 Array{Gray{N0f8},2} with eltype Gray{Normed{UInt8,8}}"

julia> summary(float32.(img))
"512×512 Array{Gray{Float32},2} with eltype Gray{Float32}"

julia> summary(Float32.(img))
"512×512 Array{Float32,2}"
```
for a color image you have
```julia

julia> summary(img)
"512×512 Array{RGB{N0f8},2} with eltype RGB{Normed{UInt8,8}}"

julia> summary(float32.(img))
"512×512 Array{RGB{Float32},2} with eltype RGB{Float32}"

julia> summary(Float32.(img))
ERROR: MethodError: no method matching Float32(::RGB{Normed{UInt8,8}})
Closest candidates are:
  Float32(::Float64) at float.jl:251
  Float32(::Int16) at float.jl:60
  Float32(::UInt16) at float.jl:66
  ...
Stacktrace:
[...]
```
That's because there's no unique way to convert a `RGB` to `Float32`. However, we may be being a bit pedantic here, because there is an [agreed-upon conversion to gray](https://stackoverflow.com/questions/687261/converting-rgb-to-grayscale-intensity), and we use it:
```julia
julia> summary(Gray{Float32}.(img))
"512×512 Array{Gray{Float32},2} with eltype Gray{Float32}"

julia> summary(Float32.(Gray{Float32}.(img)))
"512×512 Array{Float32,2}"
```

I think the reason we don't use it is because some people might expect `Float32.(img)` to do the thing that `float32.(img)` does.

Reply:
> I think the reason we don't use it is because some people might expect Float32.(img) to do the thing that float32.(img) does.

Yes, it would be awkward and newbies-unfriendly that `Float32.(rgb_img)` returns a different result from `float32.(rgb_img)` without warning. This can then be the root of many hard-to-discover bugs.